src/nvim/mbyte_defs.h

```diff
@@ -55,3 +55,13 @@ typedef struct {
   bool vc_fail;   ///< What to do with invalid characters: if true, fail,
                   ///< otherwise use '?'.
 } vimconv_T;
+
+typedef struct {
+  int value;  /// code point
+  int len;  /// length in bytes
```

Comment:
normally we don't require typed ints for these, but as there is a constraint this struct has to be equal in representation to a `uint64_t`, maybe we should use two `u/int32_t` in this case. And/or perhaps write some comment about this.